columnflow/tasks/framework/mixins.py

```diff
@@ -201,19 +201,12 @@ def calibrator_repr(self):
         """
         return str(self.calibrator_inst)
 
-    def store_parts(self) -> law.util.InsertableDict[str, str]:
+    def store_parts(self) -> law.util.InsertableDict:
         """
-        Create parts to create the output path to store intermediary results
-        for the current :py:class:`~law.task.base.Task`.
-
-        This method calls :py:meth:`store_parts` of the ``super`` class and inserts
-        `{"calibrator": "calib__{self.calibrator}"}` before keyword ``version``.
-        For more information, see e.g. :py:meth:`~columnflow.tasks.framework.base.ConfigTask.store_parts`.
-
-        :return: Updated parts to create output path to store intermediary results.
+        :return: Dictionary with parts that will be translated into an output directory path.
         """
         parts = super().store_parts()
-        parts.insert_before("version", "calibrator", f"calib__{self.calibrator_repr}")
+        parts.insert_after("config", "calibrator", f"calib__{self.calibrator_repr}")
```

Comment:
Can you add the "config" store_parts name as classatribute to the `ConfigTask`? In that way, extending to the `MultiConfigTask` will get much easier
```suggestion
        parts.insert_after(self.configs_part, "calibrator", f"calib__{self.calibrator_repr}")
```

Reply:
Not a bad idea in general. But would analyses typically change this value, or is it rather inheriting tasks that should overwrite it?

Also, I'm trying to see if there are already situations where this value is defined dynamically. If so, it would make things a bit complicated.

Reply:
it's just to make the mixins depend on the choice of `ConfigTask` vs `MultiConfigTask`, so I would not expect to change this from analysis side


(we could also just name the part "config" (or "configs") independently of single config vs multi config)

Reply:
Added in 5c6a1482.

In some custom tasks this was not generically possible, but these are perfect examples of when the inherited behavior has to be overwritten.